### PR TITLE
Update "Continue Move"  button to redirect to last page not completed

### PR DESCRIPTION
**Pull request by mkrump**
## Description

The "Continue Move" button was not redirecting to the last part completed in the HHG flow. Update to redirect to the last page that the user did not completed in the flow.

## Setup

```sh
make client_test
make e2e_test
```

1) Login to the system as a new service member
2) Create a profile
3) Create an HHG move and complete at least one part the flow (i.e through pickup address).
4) Log out
5) Log back in and click "Continue Move". You should be taken to the last page in flow that is not complete.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/163926386)

### CODE REVIEW

File: src/shared/User/ducks.js [hunk 1]
```diff
@@ -23,10 +24,14 @@ export const loadLoggedInUser = () => {
       .then(response => {
         if (response.service_member) {
           const data = normalize(response.service_member.orders, ordersArray);
+          if (data.entities.shipments) {
+            const shipmentIds = Object.keys(data.entities.shipments);
+            shipmentIds.map(id => dispatch(getShipment('Shipments.GetShipment', id)));
+          }
 
           // Only store shipments and addresses in a normalized way. This prevents
```
Comment (sarboc) [inline src/shared/User/ducks.js]: Can you update this comment since we're no longer normalizing the shipments?

Comment (mkrump) [reply src/shared/User/ducks.js]: updated.

**Comment by mkrump** [comment]
Added e2e test. Let me know what you think.

**Comment by mkrump** [comment]
@lynzt thanks for taking a look. I'm not seeing the local failure auth/logout issue. I tried just running that one test and it looked ok (it's a completely separate move, so shouldn't depend on the prev shipment test). Also, like you suggested, i added a cmd in `cypress/support/commands.js` to remove `window.fetch` for a single test. 

I wanted to make sure to address the "**NOTE**". I ran the tests locally in addition to on ci? Is there anything else that you'd suggest?

**Pull request merged**